templates/run_neo4j.java.qute

```diff
@@ -0,0 +1,31 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//JAVA 10+
+//DEPS org.neo4j.driver:neo4j-java-driver:4.3.4
+
+// NEO4J_URL="bolt://localhost" NEO4J_PASSWORD="secret"  ./{baseName}.java "MATCH (n) RETURN labels(n) AS labels, properties(n) AS props LIMIT 10"
+import static java.lang.System.*;
+import java.util.*;
+import org.neo4j.driver.*;
+
+public class {baseName} {
+    static {
+        java.util.logging.LogManager.getLogManager().reset();
```

Comment:
Why?

Reply:
Because driver logs are polluting the output

Reply:
How are they polluting the output? And why doing this with the LogManager?
If you don't want output, please configure the driver like this:

```java
try (var driver = GraphDatabase.driver(System.getenv("NEO4J_URL"),
			AuthTokens.basic( "neo4j", System.getenv("NEO4J_PASSWORD")), Config.builder().withLogging(Logging.none()).build())) {
			
		}
```

This way it's much more obvious than an arbitrary `reset`.